src/main/resources/templates/db_scripts/isbn.sql

```diff
@@ -0,0 +1,8 @@
+-- Usage: SELECT normalize_isbns(jsonb->'identifiers') FROM instance
+-- This takes each ISBN, normalizes it using RMB's normalize_digits(text),
+-- and concatenates the results using a space as separator.
+CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.normalize_isbns(jsonb_array jsonb) RETURNS text AS $$
+  SELECT string_agg(${myuniversity}_${mymodule}.normalize_digits(value->>'value'), ' ')
+  FROM jsonb_array_elements($1)
```

Comment:
Can it be changed to:
```suggestion
  FROM jsonb_array_elements($1) as value
```

for batter readability?

Reply:
Yes, name changed.